Makefile

```diff
@@ -439,3 +441,47 @@ install-kuttl:
 mocks:
 	mockery --dir=./pkg/cassandra --output=./pkg/mocks --name=ManagementApiFacade
 	mockery --dir=./pkg/reaper --output=./pkg/mocks --name=Manager  --filename=reaper_manager.go --structname=ReaperManager
+
+
+# The image base name used in GKE clusters.
+GKE_IMAGE_TAG_BASE ?= us-docker.pkg.dev/community-ecosystem/$(IMAGE_TAG_BASE)
+
+# Builds the project, builds the operator image, pushes the operator image to GKE Artifact
+# Repository, then runs e2e tests using two contexts: k8ssandra-ci-us-east and
+# k8ssandra-ci-us-north.
+# The clusters must be up and running and your ~/.kube/config file must contain the corresponding
+# contexts.
+# For security reasons, instructions for setting up the clusters are not checked in version control.
+# You must invoke this target with 2 env vars, GKE_US_EAST_IP and GKE_US_NORTH_IP containing 2
+# valid and properly configured external IPs, e.g.:
+# make gke-e2e-test GKE_US_EAST_IP=w.x.y.z GKE_US_NORTH_IP=w.x.y.z
+gke-e2e-test: build manifests
```

Comment:
This almost feels like it would be better suited as a separate script since it requires a bunch of setup, a lot of which isn't even documented.

Reply:
Indeed, there is room for improvement. For now, you can find documentation for the tags [here](https://github.com/adutra/k8ssandra-operator/blob/cloud-e2e/test/e2e/suite_test.go#L82-L128).